dev/com.ibm.ws.security.mp.jwt.1.1.config/src/com/ibm/ws/security/mp/jwt/v11/config/impl/MpConfigProxyServiceImpl.java

```diff
@@ -77,6 +79,19 @@ public <T> T getConfigValue(ClassLoader cl, String propertyName, Class<T> proper
         return null;
     }
 
+    /** return */
+    @Sensitive
+    @Override
+    public <T> List<T> getConfigValues(ClassLoader cl, Set<String> propertyNames, Class<T> propertyType) throws IllegalArgumentException {
```

Comment:
Either this has to return a Map<String, T> or it needs to take a List<String>.  Set order is not guaranteed and you don't know what order that is returned from the Set will line up with the order of the value List unless you pass in a LinkedHashSet<String> that returns the order that things are inserted into the Set.

Reply:
In the end this code is all kind of silliness.  It could be simply `Map<String, String> getConfigValues(ClassLoader cl) and it returns all of the supported ConfigProperties for the ProxyService.  That is the only thing it does.  Having all this other logic is more for testing if anything else.